deployments/k8s/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Default values for asyncapi-event-gateway.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: asyncapi/server-api
+  pullPolicy: Always
+  tag: "latest"
+
+imagePullSecrets: []
+
+replicaCount: 2
+
+# Server API opened ports.
+ports:
+  # Port where the Server API will be available.
+  http: 5000
```

Comment:
I think this is going to be `80` finally by default, right? https://github.com/asyncapi/server-api/pull/24/files#diff-841254fe75488c1bd4cd7f68f00b4be0e48dcfbc4a16b45847b68295e0e3b27bR21

Reply:
Yes, I forgot to change it :) Thanks!